packages/theme-default/src/components/Aside/index.scss

```diff
@@ -20,4 +20,5 @@
   padding: 0px 12px;
   font-size: 0.8125rem;
   line-height: 1.25rem;
+  word-break: break-all;
```

Comment:
I prefer using `overflow-wrap: break-word;`

Reply:
https://github.com/web-infra-dev/rspress/pull/1363

Reply:
got it